lib/filewatch/watch.rb

```diff
@@ -86,18 +126,18 @@ def inode(path,stat)
     def each(&block)
       synchronized do
         # Send any creates.
-        @files.keys.each do |path|
-          if ! @files[path][:create_sent]
-            if @files[path][:initial]
+        @files.each do |path, watched_value|
```

Comment:
I wanted to keep the notion of Value object, however that is not strongly communicated in the name `watch_value`.  I will go with watched_file